scripts/sysman/memoryTimestampValidBits.yml

```diff
@@ -32,10 +32,11 @@ class: $sMemory
 name: $s_mem_timestamp_bits_exp_t
 details:
     - "Number of counter bits"
+    - "1.8.1": "[DEPRECATED] No longer supported."
```

Comment:
I think we can remove this structure, since it is experimental and not used.
Please share your opinion
@AshwinKumarKulkarni @aravindksg